src/main.js

```diff
@@ -51,6 +54,27 @@ const userStore = useUserStore()
 router.isReady().then(() => {
   app.mount('#app')
   window.head = head
+
+  if (VUE_APP_SENTRY_DSN) {
+    try {
+      Sentry.init({
+        app,
+        dsn: VUE_APP_SENTRY_DSN,
+        environment: import.meta.env.MODE,
+        integrations: [
+          new BrowserTracing({
```

Comment:
```suggestion
          new Sentry.BrowserTracing({
```

Il me semble que tu peux ainsi élimiter le besoin d'une dépendance explicite à `@sentry/tracing`

Reply:
Ah oui, bien vu 🙂